AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -577,7 +577,7 @@ - (void)openURLInAuthenticationSessionWith:(NSURL *)url API_AVAILABLE(ios(11)) {
     if (!strongSelf) {
       return;
     }
-    strongSelf.authenticationSession = nil;
+    [strongSelf clearAuthenticationSession];
```

Comment:
Could we remove the try-catch in lines 595-602 with this change?

Reply:
Yes it will be removed.

Reply:
try-catch is removed.